validator/src/bin/solana-test-validator.rs

```diff
@@ -140,6 +140,12 @@ fn main() {
                 .conflicts_with("quiet")
                 .help("Log mode: stream the validator log"),
         )
+        .arg(
+            Arg::with_name("secondary-indexes")
```

Comment:
any reason not to mimic the normal validator arg here?  generally prefer consistency between the two

Reply:
No, I honestly hadn't thought of it. Just was adding a new arg. I can mimmic the normal validator, makes sense.

Reply:
+1 on Trent's suggestion